libs/sdk-ui-ext/src/dashboardView/DashboardWidgetRenderer/DashboardWidgetRenderer.tsx

```diff
@@ -76,34 +77,48 @@ export const DashboardWidgetRenderer: React.FC<IDashboardWidgetRendererProps> =
     if (isWidget(widget)) {
         if (widget.type === "insight") {
             return (
-                <DashboardItem
-                    className={cx(
-                        "type-visualization",
-                        "gd-dashboard-view-widget",
-                        getVisTypeCssClass(widget.type, visType),
-                    )}
-                    screen={screen}
-                >
-                    <DashboardItemVisualization
-                        renderHeadline={() => <DashboardItemHeadline title={widget.title} />}
-                    >
-                        {() => (
-                            <InsightRenderer
-                                insight={insight}
-                                insightWidget={widget}
-                                backend={backend}
-                                workspace={workspace}
-                                filters={filters}
-                                filterContext={filterContext}
-                                drillableItems={drillableItems}
-                                onDrill={onDrill}
-                                onError={onError}
-                                ErrorComponent={ErrorComponent}
-                                LoadingComponent={LoadingComponent}
-                            />
-                        )}
-                    </DashboardItemVisualization>
-                </DashboardItem>
+                <Measure client>
+                    {({ measureRef, contentRect }) => {
+                        return (
+                            <div style={{ height: "100%", width: "100%" }} ref={measureRef}>
```

Comment:
Please make the `{ height: "100%", width: "100%" }` style a constant and use it here. This will improve the performance by avoiding unnecessary re-renders of the div.